homeassistant/components/boschshc/binary_sensor.py

```diff
@@ -0,0 +1,58 @@
+"""Platform for binarysensor integration."""
+import logging
+
+from boschshcpy import SHCSession, SHCShutterContact
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_DOOR,
+    DEVICE_CLASS_WINDOW,
+    BinarySensorEntity,
+)
+from homeassistant.const import CONF_IP_ADDRESS
+
+from .const import DOMAIN
+from .entity import SHCEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the SHC binary sensor platform."""
+    device = []
+    session: SHCSession = hass.data[DOMAIN][config_entry.entry_id]
+
+    for binarysensor in session.device_helper.shutter_contacts:
+        device.append(
+            ShutterContactSensor(
+                device=binarysensor,
+                room_name=session.room(binarysensor.room_id).name,
+                controller_ip=config_entry.data[CONF_IP_ADDRESS],
+            )
+        )
+
+    if device:
+        async_add_entities(device)
+
+
+class ShutterContactSensor(SHCEntity, BinarySensorEntity):
+    """Representation of a SHC shutter contact sensor."""
+
+    @property
+    def is_on(self):
+        """Return the state of the sensor."""
+        if self._device.state == SHCShutterContact.ShutterContactService.State.OPEN:
+            return True
+        if self._device.state == SHCShutterContact.ShutterContactService.State.CLOSED:
+            return False
+        return None
```

Comment:
We can simplify this.
```suggestion
        return self._device.state == SHCShutterContact.ShutterContactService.State.OPEN
```